supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkClusterProcess.scala

```diff
@@ -0,0 +1,173 @@
+package cromwell.backend.impl.spark
+
+import java.nio.file.Path
+
+import akka.actor.ActorSystem
+import cromwell.backend.impl.spark.SparkClusterProcess.{ParserResponse, TerminalStatus}
+import spray.http.{HttpRequest, HttpResponse, StatusCodes}
+import spray.json.{DefaultJsonProtocol, JsonParser}
+import spray.client.pipelining._
+
+import scala.concurrent.{ExecutionContext, Future, Promise}
+import better.files._
+import com.typesafe.scalalogging.Logger
+import org.slf4j.LoggerFactory
+import spray.httpx.unmarshalling._
+
+import scala.util.{Failure, Success, Try}
+
+object SparkClusterProcess {
+  sealed trait SparkClusterResponse
+  case class SuccessResponse(action: String, driverState: String, serverSparkVersion: String,
+                             submissionId: String, success: Boolean, workerHostPort: String, workerId: String) extends SparkClusterResponse
+  sealed trait ParserResponse
+  case class SuccessfulParsedResponse(action: String, message: String, serverSparkVersion: String, submissionId: String, success: Boolean) extends ParserResponse
+
+  sealed trait TerminalStatus
+  case class Failed(error: Throwable) extends TerminalStatus
+  case object Finished extends TerminalStatus
+
+  object SparkClusterJsonProtocol extends DefaultJsonProtocol {
+    implicit val sparkStatusResponseFormat = jsonFormat7(SuccessResponse)
+    implicit val successfulParsedResponseFormat = jsonFormat5(SuccessfulParsedResponse)
+  }
+}
+
+object SparkClusterConstants {
+  lazy val SparkClusterMasterDefault = "spark-master"
```

Comment:
Are all of these constants being used in multiple places? If anything is being used in only a single place, de-constantize